src/keys/rsa.js

```
@@ -127,9 +127,39 @@ function unmarshalRsaPublicKey (bytes) {
   return new RsaPublicKey(key)
 }
 
-function generateKeyPair (bits) {
-  const p = rsa.generateKeyPair({bits})
-  return new RsaPrivateKey(p.privateKey, p.publicKey)
+function fastKeys (bits, cb) {
+  if (typeof window === 'undefined') {
+    let ursa
+    try {
+      ursa = require('ursa')
+    } catch (err) {
```

Ah, did not know is what an optional dependency and my spidey senses goes off when I see a empty catch in a try/catch block.